server/locales/de.json

```diff
@@ -1,25 +1,28 @@
-  "error server": "Ein interner Fehler ist aufgetreten.",
-  "error bad credentials": "Ungültiges Passwort.",
-  "error keys not initialized": "Die Schlüssel sind nicht initialisiert.",
-  "error login failed": "Anmeldung fehlgeschlagen.",
-  "reset password email from": "Ihr Cozy <no-reply@%{domain}>",
-  "reset password email subject": "[Cozy] Rücksetzen Sie Ihr Passwort",
-  "reset password email text": "Es scheint als ob Sie das Passwort für Ihr Cozy vergessen haben.\nKeine Sorge; folgen Sie einfach dem Link unten um ein neues zu erstellen:\n\nhttps://%{domain}/password/reset/%{key}\n\nVergessen danach Sie nicht all Ihre verschlüsselten Daten zu aktualiseren!"
+    "error title": "Ups, ein Fehler ist aufgetreten",
+    "error headline": "Es scheint als ob etwas schief gelaufen ist.",
+    "error reinsurance": "Keine Bange, wahrscheinlich ist es nicht so schlimm!",
+    "error temporary issue": "Das Problem sollte vorübergehend sein; bitte probieren Sie es in 5 Minuten noch einmal.",
+    "error try restart": "Sollte dannach sich trotzdem nichts ändern, probieren Sie Ihr Cozy neu zu starten.",
+    "error contact cozy team": "Wenn das Problem weiterhin besteht, scheuen Sie sich nicht das Cozy team zu kontaktieren:",
+    "error contact forum": "Fragen Sie nach Hilfe auf <a href=\"https://forum.cozy.io\">our forum</a>",
+    "error contact email": "Senden Sie eine E-Mail an contact@cozycloud.cc",
+    "error contact irc": "Berichten Sie den Sachverhalt on IRC (=Internet relay chat), #cozycloud on irc.freenode.net",
+    "error wait a bit": "Für 5 Minuten warten",
+    "error restart app": "Starten Sie die App neu (\"App Management\" Menü)",
+    "error reinstall app": "App deinstallieren",
+    "error not found info": "Sie versuchen auf eine App zu zugreifen die entweder nicht installiert ist oder im Moment installiert wird.",
+    "error try to fix": "Sie können die folgenden Schritte verwenden und versuchen das Problem zu lösen:",
+    "error contact developer": "Wenn nichts funktioniert hat, können Sie den Entwickler der App oder das Cozy Team kontaktieren:",
+    "error public info": "Bitte warten Sie für 5 Minuten, sollte dann nichts passiert sein, kontaktieren Sie den Cozy Besitzer!",
+    "error server": "Ein interner Fehler ist aufgetreten.",
+    "error bad credentials": "Ungültiges Passwort.",
+    "error keys not initialized": "Die Schlüssel sind nicht initialisiert.",
+    "error login failed": "Anmeldung fehlgeschlagen.",
+    "reset password email from": "Ihr Cozy <no-reply@%{domain}>",
+    "reset password email subject": "[Cozy] Rücksetzen Sie Ihr Passwort",
+    "reset password email text": "Es scheint als ob Sie das Passwort für Ihr Cozy vergessen haben.\nKeine Sorge; folgen Sie einfach dem Link unten um ein neues zu erstellen:\n\nhttps://%{domain}/password/reset/%{key}\n\nVergessen danach Sie nicht all Ihre verschlüsselten Daten zu aktualiseren!"
```

Comment:
weird, something went wrong during Transifex merge. But how did you manage to find this one ???